lib/commons/loading.dart

```diff
@@ -6,19 +6,32 @@ import 'package:flutter_spinkit/flutter_spinkit.dart';
 class Loading extends StatelessWidget {
 
   final double size;
+  final Widget loader;
 
-  Loading({this.size = 30});
+  Loading({this.size = 30, this.loader});
```

Comment:
As you defined a factory method, couldn't this constructor be private? And you can also add another factory method `Loading.chasingDots`